airbyte-server/src/test/java/io/airbyte/server/handlers/ConnectionsHandlerTest.java

```diff
@@ -612,6 +619,321 @@ void testEnumConversion() {
       assertTrue(Enums.isCompatible(NamespaceDefinitionType.class, io.airbyte.config.JobSyncConfig.NamespaceDefinitionType.class));
     }
 
+    @Test
+    void testNoDiff() {
```

Comment:
It would be a bit more readable if these configuration diff tests were grouped together (e.g. in a nested test class), or at least had the same prefix (`testConfigurationDiff...`)

Reply:
Done